future-apps/payroll/test/contracts/Payroll_rates.test.js

```diff
@@ -55,13 +55,18 @@ contract('Payroll rates handling,', ([owner, employee, anyone]) => {
         const previousDAI = await DAI.balanceOf(employee)
         const previousANT = await ANT.balanceOf(employee)
 
-        const { tx, receipt } = await payroll.payday(PAYMENT_TYPES.PAYROLL, 0, { from: employee })
+        const { tx, receipt, logs } = await payroll.payday(PAYMENT_TYPES.PAYROLL, 0, { from: employee })
         const { gasPrice } = await web3.eth.getTransaction(tx)
         const txCost = gasPrice.mul(receipt.gasUsed)
 
+        const events = logs.filter(l => l.event === 'SendPayment')
```

Comment:
Nit: I think you can use `getEvents(receipt, 'SendPayment')` from `../helpers/events'`

Reply:
Done in 1b59642